webapp/components/test-file-results-table.js

```diff
@@ -184,6 +214,23 @@ class TestFileResultsTable extends WPTColors(PathInfo(TestRunsBase)) {
         type: Boolean,
         value: false,
       },
+      selectedMetadata: {
+        type: Array,
+        value: [],
+        observer: 'clearSelectedCells',
+      },
+      selectedCells: {
+        type: Array,
+        value: [],
+      },
+      testPlural: {
+        type: String,
+        computed: 'computeTestPlural(selectedMetadata)',
+      },
+      isTriageMode: {
+        type: Boolean,
+        observer: 'isTriageModeUpdated',
+      },
```

Comment:
How many of these new properties and methods are the same as the ones in `wpt-results.js`? Could we create a mixin?